frontend/src/components/WritingTable/WritingTable.tsx

```diff
@@ -0,0 +1,101 @@
+import { MediumLogoIcon, TistoryLogoIcon } from 'assets/icons';
+import { usePageNavigate } from 'hooks/usePageNavigate';
+import { Fragment } from 'react';
+import { styled } from 'styled-components';
+import { Writing } from 'types/apis/writings';
+import { Blog } from 'types/domain';
+import { dateFormatter } from 'utils/date';
+
+type Props = { writings: Writing[] };
+
+const WritingTable = ({ writings }: Props) => {
+  const { goWritingPage } = usePageNavigate();
+
+  const getPublishedToLogoIcon = (blogName: Blog) => {
+    switch (blogName) {
+      case 'MEDIUM':
+        return <MediumLogoIcon width='2.4rem' height='2.4rem' />;
+      case 'TISTORY':
+        return <TistoryLogoIcon width='2.4rem' height='2.4rem' />;
+    }
+  };
+
+  return (
+    <>
+      <S.WritingTableContainer>
+        <colgroup>
+          <col style={{ width: '60%' }} />
+          <col style={{ width: '20%' }} />
+          <col style={{ width: '20%' }} />
```

Comment:
```suggestion
          <col width='60%' />
          <col width='20%' />
          <col width='20%' />
```
이거 col 기본 속성에 width가 있는데 이렇게 쓰는 것은 어떨까요?

Reply:
오 이건 몰랐네요.
반영했습니다!